tests/functional/features/navigation.js

```diff
@@ -24,3 +25,27 @@ test(`On the home page, click to navigate to the login/register page`, async t =
 
   await t.expect(actual).eql(expected, should)
 })
+
+test(`Navigating to ${pages.TOKEN} without logging in redirects to ${pages.LOGIN}`, async t => {
+  const should = `Should load the login page at ${pages.LOGIN}`
+  const expected = pages.LOGIN
+
+  await t.navigateTo(`${SITE}${pages.TOKEN}`)
+  await LoginRegisterPage.pageClass()
+  const actual = await getLocation('pathname')
+
+  await t.expect(actual).eql(expected, should)
+})
+
+test(`A logged in user can access the api token page at ${pages.TOKEN}`, async t => {
+  const should = `Should load the token page at ${pages.TOKEN}`
+  const expected = pages.TOKEN
+  await t
+    .useRole(LoginRegisterPage.frostUser('fah@test.com', 'Foo@foo12345'))
```

Comment:
This is showing that the "specified resource does not exist" error message when logging in for me.

Reply:
Ah, you are right. That's an account I created locally for testing. I'll need to figure out a way to setup a user account.